### PR TITLE
[1LP][RFR] Adding Version Pick to handle Ansible Repo Add operation Flash Message

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Fixing__ Flash message with version pick since developers changed the flash message in 5.9.2.0

{{pytest:  cfme/tests/ansible/test_embedded_ansible_services.py --long-running -s  -k 'test_service_ansible_playbook_crud'}}

### CODE REVIEW

File: cfme/ansible/repositories.py [hunk 1]
```diff
@@ -226,8 +227,11 @@ def create(self, name, url, description=None, scm_credentials=None, scm_branch=N
         repo_list_page = self.create_view(RepositoryAllView)
         assert repo_list_page.is_displayed
         repo_list_page.flash.assert_no_error()
-        repo_list_page.flash.assert_message(
-            'Add of Repository "{}" was successfully initialized.'.format(name))
+        initiated_message = VersionPick({
+            Version.lowest(): 'Add of Repository "{}" was successfully initialized.'.format(name),
+            '5.9.2': 'Add of Repository "{}" was successfully initiated.'.format(name),
+        }).pick(self.browser.product_version)
```
Comment (quarckster) [inline cfme/ansible/repositories.py]: **REQUIRED**
```python
if self.appliance.version < "5.9.2":
    initiated_message = 'Add of Repository "{}" was successfully initialized.'.format(name)
else:
    initiated_message = 'Add of Repository "{}" was successfully initiated.'.format(name)
```

Comment (kedark3) [reply cfme/ansible/repositories.py]: @quarckster why can't we use VersionPick?

Comment (quarckster) [reply cfme/ansible/repositories.py]: Because `VersionPick` is supposed to be used only on views.

Comment (kedark3) [reply cfme/ansible/repositories.py]: @quarckster  updated.

**Comment by psav** [comment]
Rebase needed pls

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by kedark3** [comment]
@psav I did rebase. Kindly approve :)

**Comment by kedark3** [comment]
Failures seem to be due to sprout malfunction.

**Comment by sshveta** [comment]
This got fixed here https://github.com/ManageIQ/integration_tests/pull/6967.
Wonder why it's not asking to rebase

**Pull request merged**